### PR TITLE
Add usage instructions to the readme

### PR DESCRIPTION
**Pull request by M1ke**
I had to do various bits of Googling to be able to use this generator; thought I'd contribute by adding some basic instructions and avoiding pitfalls (e.g unsupported node version for the Yeoman version)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,30 @@ in question).
...
 
+## Usage
+
+This project requires [Yeoman](https://yeoman.io). Install globally with:
+
+```
+npm install -g yo
+```
+
+_NB as of 2020-03-05 this requiest node >= v10_
```
Comment (dancrumb) [inline README.md]: Small typo here

File: README.md [hunk 2]
```diff
@@ -10,6 +10,30 @@ in question).
...
+npm install -y generator-serverless-policy
+```
+
+Now run in the directory in which you wish to output the policy JSON file:
+
+```
+yo serverless-policy
+```
+
+The generator will ask questions about region, stage, DynamoDB and S3 before outputting a policy. If this is the first time you have used Yeoman the tool will prompt you about sharing analytics with them; _the answer to this does not affect the generation of the policy_.
```
Comment (dancrumb) [inline README.md]: Could you clarify that it is yeoman asking for analytics and not this generator, please?

**Review by dancrumb — CHANGES_REQUESTED** [review]
Thanks for the updates. Couple of small tweaks and we're good to go

**Comment by M1ke** [comment]
Done those changes

**Comment by dancrumb** [comment]
Thanks!

**Pull request merged**